.github/workflows/elixir.yml

```diff
@@ -127,9 +127,17 @@ jobs:
         image: amazon/dynamodb-local:2.0.0
         ports:
           - 8000:8000
+      mosquitto:
+        image: eclipse-mosquitto:2.0
+        ports:
+          - 1883:1883
+        volumes:
+          - ./mosquitto:/mosquitto/config/
```

Comment:
question: any reason this is different from what's in `docker-compose.yml`?

Reply:
That was a mistake, should be fixed now.